quickstart_lists/README.md

```diff
@@ -0,0 +1,26 @@
+This directory contains 'starter lists' of packages for bootstrapping and syncing on-prem-builders.
```

Comment:
"Quickstart" is new terminology - @apriofrost or @kagarmoe may want to weigh in on what to call these.